cmd/core/service/ipam.go

```diff
@@ -0,0 +1,127 @@
+package service
+
+import (
+	"errors"
+	"net"
+	"strconv"
+	"sync"
+)
+
+type IPAM struct {
```

Comment:
I'd recommend you to devide this IPAM struct to actual IPAM(IP address management) entity and F-TEID management entity.